topics/assembly/tutorials/largegenome/tutorial.md

```diff
@@ -0,0 +1,587 @@
+
+Look at the Fasta Statistics output files for the status of the assemblies after each polishing tool. From these, we can see that some polishes decrease or increase the size of the assembly. The final size is approximately 300,000 bp shorter than the original Flye assembly. (These numbers may vary slightly even if the same input data is used). 
+
+# Genome quality assessment
+
+The polished genome is in 145 contigs with a total length of ~ 9.6 million base pairs.  We have some idea of how these contigs may be joined (or not) from the Bandage assembly graph. 
+
+> ### {% icon question %} Questions
+>
+> 1. How good is the assembly?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1.  One measure is the N50, a number that indicates how large the contigs are (although, have the contigs been joined correctly)? Another measure is to see if expected gene sequences are found in the assembly, using a tool called BUSCO. For a discussion on these and other methods, see {% cite Wang2020%}.
+> >
+> {: .solution}
+{: .question}
+
+Here, we will use the BUSCO tool to annotate the genome and then assess whether expected genes are found. Note: this is a brief annotation only, not the full genome annotation that would typically be done following genome assembly. 
+
+For more about Busco see {% cite Simo2015 %}.
+
+Genome assessment:
+
+!["Image of genome assessment, showing contigs as bold bars, with one contig annotated (various sections labelled by coloured boxes), and then compared to an expected annotation."](images/assess.png){: width="650"}
+
+We will also map the assembled contigs to a known reference genome using the tool Quast, to see how they align. For more about quast see [the manual](http://quast.sourceforge.net/docs/manual.html). 
+
+For more about the Icarus browser see {% cite Mikheenko2016 %}.
+
+*Genome assessment workflow*
+
+> ### {% icon hands_on %} Hands-on: Run the Genome Assessment workflow
+>* Find this workflow, enter the correct input files, and run.
+* What it does: 
+  * Assesses the quality of the genome assembly: generate some statistics and determine if expected genes are present
+  * Align contigs to a reference genome.
+* Inputs: 
+  * polished assembly
+  * reference_genome.fasta (e.g. of a closely-related species, if available). 
+* Outputs
+  * Busco table of genes found
+  * Quast HTML report, and link to Icarus contigs browser,  showing contigs aligned to a reference genome
+* Tools used: 
+  * Busco
+  * Quast
+* Input parameters: None required
+* Workflow steps:
+  * Polished assembly => Busco:
+  * First: predict genes in the assembly: using Metaeuk
+  * Second: compare the set of predicted genes to the set of expected genes in a particular lineage. Default setting for lineage: Eukaryota
+  * Polished assembly and a reference genome => Quast:
+  * For the tutorial we will use the Arabidopsis genome. This is not closely related to our Eucalyptus species but will give an idea of how to use Quast. 
```

Comment:
I'm not sure if this is a good idea; why do not assemble the *Arabidopsis* genome from the beginning?

Reply:
Just using Arabidopsis as an example - wanting to get users comfortable with using their own data and then seeking their own appropriate comparison genome.